Frameworks/CoreText/DWriteWrapper.mm

```diff
@@ -363,9 +364,10 @@ HRESULT STDMETHODCALLTYPE DrawGlyphRun(_In_ void* clientDrawingContext,
         _DWriteGlyphRunDescription glyphRunDescriptionInfo;
         glyphRunDescriptionInfo._stringLength = glyphRunDescription->stringLength;
         glyphRunDescriptionInfo._textPosition = glyphRunDescription->textPosition;
-        std::copy(glyphRunDescription->clusterMap,
-                  glyphRunDescription->clusterMap + glyphRun->glyphCount,
-                  std::back_inserter(glyphRunDescriptionInfo._clusterMap));
+        std::transform(glyphRunDescription->clusterMap,
+                       glyphRunDescription->clusterMap + glyphRun->glyphCount,
+                       std::back_inserter(glyphRunDescriptionInfo._clusterMap),
+                       [&](uint16_t index) { return index + glyphRunDescription->textPosition; });
```

Comment:
>& [](start = 24, length = 1)

scope this down to just what you need (global captures are dangerous, looks like you just need glyphRunDescription?)